biosimulations/libs/dispatch/api-models/src/lib/common/dispatch-simulation.model.ts

```diff
@@ -0,0 +1,59 @@
+import { prop } from '@typegoose/typegoose';
```

Comment:
@gmarupilla Is this needed in this library, or should the dispatch/datamodel one be used?

Reply:
removed `dispatch/datamodel`